Source/WebCore/page/DebugPageOverlays.cpp

```diff
@@ -259,9 +264,12 @@ class InteractionRegionOverlay final : public RegionOverlay {
 
     FloatRect rectForSettingAtIndex(unsigned);
     bool valueForSetting(ASCIILiteral);
-        
+
+    std::optional<std::pair<RenderLayer&, GraphicsLayer&>> activeLayer();
```

Comment:
Can this be `const`? (Also applies to several other member functions in this class).